code/modules/projectiles/guns/ballistic.dm

```diff
@@ -110,6 +110,15 @@
 	var/feedback_recoil_reverse = FALSE // TRUE for clockwise , FALSE for anti-clockwise
 	var/feedback_slide_close_move = TRUE // does the slide closing cause the gun to twist clockwise?
 
+	available_attachments = list(
+		/obj/item/attachment/scope/simple,
```

Comment:
Is there a reason this isn't compacted into /scope for both of them?

Reply:
readability? not every scope is guaranteed to fit on every rifle? not allowing the non-functional sight base to be mounted?